pht_federated/aggregator/api/crud/crud_discovery.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy.orm import Session
+from fastapi import Depends
+from .base import CRUDBase, CreateSchemaType, ModelType, Optional
+from fastapi.encoders import jsonable_encoder
+from pht_federated.aggregator.api.models.discovery import DiscoverySummary
+from pht_federated.aggregator.api.schemas.discovery import SummaryCreate, SummaryUpdate
+from .. import dependencies
+
+
+class CRUDDiscoveries(CRUDBase[DiscoverySummary, SummaryCreate, SummaryUpdate]):
+
+    def get_by_discovery_id(self, proposal_id: int, db: Session = Depends(dependencies.get_db)) -> DiscoverySummary:
+        discovery = db.query(DiscoverySummary).filter(DiscoverySummary.proposal_id == proposal_id).first()
+        return discovery
```

Comment:
there can be mutltiple discoveries under a proposal id right? so this should return a list?